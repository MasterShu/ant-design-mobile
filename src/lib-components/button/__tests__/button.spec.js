import Button from '..'
import { mount } from '@vue/test-utils'

describe('Button', () => {
  it('renders correctly', () => {
    const wrapper = mount({
      render () {
        return <Button>First Button</Button>
      }
    })
    expect(wrapper.html()).toMatchSnapshot()
  })

  it('create primary button', () => {
    const wrapper = mount({
      render () {
        return <Button type='primary'>Primary</Button>
      }
    })
    expect(wrapper.contains('.antd-btn-primary')).toBe(true)
  })

  it('renders Chinese characters correctly', () => {

  })
})