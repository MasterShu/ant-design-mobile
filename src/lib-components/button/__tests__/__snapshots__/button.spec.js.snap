// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button render 1`] = `<div><button></button></div>`;

exports[`Button renders correctly 1`] = `<div><button>First Button</button></div>`;
